nntrainer/utils/node_exporter.h

```diff
@@ -3,7 +3,7 @@
  * Copyright (C) 2021 Jihoon Lee <jhoon.it.lee@samsung.com>
  *
  * @file node_exporter.h
- * @date 08 April 2021
+ * @date 09 April 2021
```

Comment:
is this intended?

Reply:
yes.

Reply:
It was 09 April but I just copy and pasted the doc and fixed it later lol

Reply:
:)